index.html

```diff
@@ -17,70 +17,84 @@ <h2 class="h1__subtitle">Sail Away</h2>
       <section class="questioncard__box">
         <img
```

Comment:
Maybe it should be better to wrap a `button` element around the image tag for semantic reasons?